n3fit/src/n3fit/tests/regressions/quickcard_2.json

```diff
@@ -49,8 +49,8 @@
       "trainable": true
     }
   ],
-  "stop_epoch": 313,
-  "best_epoch": 11,
+  "stop_epoch": 314,
+  "best_epoch": 12,
```

Comment:
I saw that you didn't know some time ago, but do you have any idea now why do you have to change this?

Reply:
Yes, I'm also confused about this one. My only guess is that the change in conda packages means a slightly different version of numpy or something (because if it were different random numbers one would expect a bigger change!)